rtc-video-quality/generate_graphs.py

```diff
@@ -94,9 +169,12 @@ def generate_graphs(output_dict, graph_data, target_metric,
 def main():
     args = parser.parse_args()
     graph_data = []
+    generate_images = False
     for f in args.graph_files:
         graph_data += ast.literal_eval(f.read())
-
+    generate_stt(graph_data, args.out_dir)
+    if not generate_images:
```

Comment:
As far as I can tell, generate_images is always false here. If so, please put a comment above generate_images to let users know that it's a constant.